src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier supplier = new FigureSupplier();
+        for (int i = 0; i < 6; i++) {
+            if (i < 3) {
+                supplier.getRandomFigure().draw();
+                continue;
+            }
+            supplier.getDefaultFigure().draw();
```

Comment:
Don't return `null` from a method. Ensure that `getDefaultFigure()` method in `FigureSupplier` class doesn't return `null`. If there's a `switch case` in `getDefaultFigure()`, handle the default case properly.